mappings/net/minecraft/util/io/CsvWriter.mapping

```diff
@@ -1,17 +1,19 @@
-CLASS net/minecraft/class_4456 net/minecraft/util/CsvWriter
+CLASS net/minecraft/class_4456 net/minecraft/util/io/CsvWriter
```

Comment:
Part of the debug command outputs, this should really go there or with profiler than as a general IO utility

Reply:
Only usages are in `net/minecraft/server/world` for exporting chunk data, and it's not really a profiler class, but whatever.